tests/integration/test_data_type_roundtrips.py

```diff
@@ -65,3 +65,11 @@ def test_none_roundtrip(client):
     test = fql("${none}", none=none)
     result = client.query(test).data
     assert result == none
+
+
+# def test_document_roundtrip(client, a_collection):
+#     test = client.query(
+#         fql("${col}.create({'name':'Scout'})", col=a_collection))
+#     assert type(test.data) == Document
+#     result = client.query(fql("${doc}", doc=test.data))
+#     assert test.data == result.data
```

Comment:
This passes on the current build, but I'm intentionally leaving it commented out. I found a 3.8 incompatibility issue that was mostly luck—the first failed test reports, the rest get cancelled—so decided to ensure all tests pass for now.